### PR TITLE
Fixes Subscription entity delete upload manifest

### PR DESCRIPTION
**Pull request by ldjebran**
Test output at https://github.com/SatelliteQE/robottelo/pull/6708

### CODE REVIEW

File: airgun/widgets.py [hunk 1]
```diff
@@ -562,8 +562,20 @@ def messages(self):
...
+            ignore_messages = []
+        self.logger.info('asserting there are no error messages')
+        for message in self.messages:
+            message_text = message.text
+            if message.type not in {'success', 'info', 'warning'}:
+                if message_text in ignore_messages:
+                    self.logger.info('ERROR MESSAGE IGNORED %s: %r', message.type, message_text)
+                    continue
+                self.logger.error('%s: %r', message.type, message_text)
+                raise AssertionError('assert_no_error: {}: {}'.format(message.type, message_text))
```
Comment (mirzal) [inline airgun/widgets.py]: I know this is modified version of code in widgetastic_patternfly and they aren't consistent themselves, but I would slightly prefer us to use one form of formatting inside function.

`self.logger.error('{}: {!r}'.format(message.type, message_text))` should have the same effect.

File: airgun/entities/subscription.py [hunk 2]
```diff
@@ -13,14 +13,17 @@
...
+                                    ignore_error_messages=None):
         """Helper ensuring that task (upload / delete manifest / subscription)
         has finished. Run after action invoking task to leave Satellite
         in usable state.
         Currently waits for three events. Since page is written asynchronously,
         they can happen in any order.
         :param name: Name of running task
         :param has_manifest: Should manifest exist after task ended?
+        :param timeout: Waiting timeout
+        :param ignore_error_messages: Error messages to ignore
```
Comment (mirzal) [inline airgun/entities/subscription.py]: Could you elaborate a bit what data structure is expected here? Maybe "List of strings with error messages to ignore; exact match will be used"?

File: airgun/entities/subscription.py [hunk 3]
```diff
@@ -160,10 +166,6 @@ class SubscriptionList(SubscriptionNavigationStep):
 
     def pre_navigate(self, _tries, *args, **kwargs):
         super(SubscriptionList, self).pre_navigate(_tries, *args, **kwargs)
-        wait_for(
-                lambda: not self.view.fake_fade_widget.is_displayed,
-                handle_exception=True, logger=self.view.logger, timeout=10 * 60
-        )
```
Comment (mirzal) [inline airgun/entities/subscription.py]: If it's safe to remove this `wait_for`, do we still need `pre_navigate` at all?

Comment (ldjebran) [reply airgun/entities/subscription.py]: Yes this safe run the test many time no problem faced, even was not able to find fade elelemt

**Comment by ldjebran** [comment]
test output for the latest changes
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_subscription.py::test_positive_end_to_end 
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-4.0.2, py-1.5.4, pluggy-0.8.0 -- /home/dlezz/.pyenv/versions/robottelo/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: redis
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.26.0, timeout-1.3.3, services-1.3.1, mock-1.10.0, forked-0.2, cov-2.6.1
collecting ... 2019-02-04 17:38:14 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                             

tests/foreman/ui_airgun/test_subscription.py::test_positive_end_to_end PASSED                          [100%]

========================================= 1 passed in 465.53 seconds =========================================

```

File: airgun/entities/subscription.py (hunk 3 above)
Comment (ldjebran) [reply airgun/entities/subscription.py]: yes right no need for pre_navigate function , thanks to point it

**Comment by abalakh** [comment]
Merging as robottelo PR got merged already and there's 2 acks (assuming `looks good to me` == `ack`).

**Pull request merged**